README.md

```diff
@@ -1,26 +1,40 @@
 # Service Composition
 
-This guide walks you through the process of implementing a service composition using Ballerina language. A service composition is an aggregate of services collectively composed to automate a particular task or business process. 
+A service composition is an aggregate of services collectively composed to automate a particular task or business process. 
+
+> This guide walks you through the process of implementing a service composition using Ballerina language. 
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-you-build)
+- [Prerequisites](#pre-req)
+- [Developing the service](#developing-service)
+- [Testing](#testing)
+- [Deployment](#deploying-the-scenario)
+- [Observability](#observability)
 
 ## <a name="what-you-build"></a>  What you’ll build
-To understanding how you can build a service composition using Ballerina, let's consider a real-world use case of a Travel agency that arranges complete tours for users. A tour package includes Airline ticket reservation, Hotel room reservation and Car rental. Therefore, the Travel agency service requires communicating with other necessary back-ends. The below diagram illustrates this use case clearly.
+To understanding how you can build a service composition using Ballerina, let's consider a real-world use case of a Travel agency that arranges complete tours for users. A tour package includes airline ticket reservation, hotel room reservation and car rental. Therefore, the Travel agency service requires communicating with other necessary back-ends. The following diagram illustrates this use case clearly.
 
-![alt text](https://github.com/ballerina-guides/service-composition/blob/master/images/serviceComposition.png)
+![alt text](/images/service_composition.png)
+
+Travel agency is the service that acts as the composition initiator. The other three services are external services that the travel agency service calls to do airline ticket booking, hotel reservation and car rental. These are not necessarily Ballerina services and can theoretically be third-party services that the travel agency service calls to get things done. However, for the purposes of setting up this scenario and illustrating it in this guide, these third=party services are also written in Ballerina.
```

Comment:
Can you please change "third=party" into third-party?

Reply:
Done. Please check now.